docs/API.md

```diff
@@ -163,15 +163,15 @@ __Parameters__
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`bucketName`  | _string_  |name of the bucket.   |
+|`bucketName`  | _string_  |Name of the bucket |
 
 
 __Return Values__
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`found`  | _bool_ | indicates whether bucket exists or not  |
-|`err` | _error_  | standard error  |
+|`found`  | _bool_ | Indicates whether bucket exists or not  |
+|`err` | _error_  | Standard error  |
```

Comment:
"Standard Error" consistent with below changes.. or change the below changes to "Standard error"